images/linux/scripts/tests/Java.Tests.ps1

```diff
@@ -0,0 +1,47 @@
+Import-Module "$PSScriptRoot/../helpers/Common.Helpers.psm1" -DisableNameChecking
+
+Describe "Java" {
+
+	if (Test-IsUbuntu20) {
+        $DefaultJavaVersion = "11"
+    }
+
+    if ((Test-IsUbuntu16) -or (Test-IsUbuntu18)) {
+        $DefaultJavaVersion = "8"
+    }
+
+    It "Java <DefaultJavaVersion> is default" -TestCases @{ DefaultJavaVersion = $DefaultJavaVersion } {
+        $actualJavaPath = bash -c "source ${env:INSTALLER_SCRIPT_FOLDER}/../helpers/etc-environment.sh && getEtcEnvironmentVariable JAVA_HOME"
+        $expectedJavaPath = bash -c "source ${env:INSTALLER_SCRIPT_FOLDER}/../helpers/etc-environment.sh && getEtcEnvironmentVariable JAVA_HOME_${DefaultJavaVersion}_X64"
+
+        $actualJavaPath | Should -Not -BeNullOrEmpty
+        $expectedJavaPath | Should -Not -BeNullOrEmpty
+        $actualJavaPath | Should -Be $expectedJavaPath
+    }
+
+    It "<ToolName>" -TestCases @(
+        @{ ToolName = "java" }
+        @{ ToolName = "mvn" }
+        @{ ToolName = "ant" }
+        @{ ToolName = "gradle" }
+    ) {
+        "$ToolName -version" | Should -ReturnZeroExitCode
+    }
+
+    It "Java <Version>" -TestCases @(
+        @{ Version = "1.7" }
```

Comment:
I think we should move the list of Java versions to toolset for Ubuntu and Windows images to avoid hardcode. It is out of scope for this PR so I have created low priority item on internal board.
https://github.com/actions/virtual-environments-internal/issues/1635
cc: @AlenaSviridenko